src/bootstrap/lib.rs

```diff
@@ -307,16 +307,20 @@ impl Crate {
 #[derive(Debug, Hash, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
 pub enum Mode {
     /// Build the standard library, placing output in the "stageN-std" directory.
-    Libstd,
+    Std,
 
     /// Build libtest, placing output in the "stageN-test" directory.
-    Libtest,
+    Test,
 
-    /// Build librustc and compiler libraries, placing output in the "stageN-rustc" directory.
-    Librustc,
+    /// Build librustc, codegen and compiler libraries, placing output
+    /// in the "stageN-rustc" directory.
```

Comment:
This comment should be modified for Codegen since we place its output in -codegen directory.